csm/conf/post_install.py

```diff
@@ -183,10 +210,36 @@ def _configure_rsyslog(self):
         os.makedirs(const.RSYSLOG_DIR, exist_ok=True)
         if os.path.exists(const.RSYSLOG_DIR):
             Setup._run_cmd(f"cp -f {const.SOURCE_RSYSLOG_PATH} {const.RSYSLOG_PATH}")
+            csm_setup_log_path = self._get_log_path_from_conf_store + "/csm_setup.log"
```

Comment:
csm_setup.log will be at /tmp/ location i guess... even in k8s environment..Please reconfirm